src/main/resources/i18n/strings.properties

```diff
@@ -336,6 +336,15 @@ preferences.contribute.registeredFor=Supporter certificate registered for %s
 preferences.contribute.noCertificate=Support Cryptomator and receive a supporter certificate. It's like a license key but for awesome people using free software. ;-)
 preferences.contribute.getCertificate=Don't have one already? Learn how you can obtain it.
 preferences.contribute.promptText=Paste supporter certificate code here
+preferences.contribute.thankYou=Thank you for supporting Cryptomator’s open-source development!
+preferences.contribute.donate=Donate
+preferences.contribute.sponsors=Sponsors
+
+### Remove License Key Dialog
+removeCert.title=Remove Cert
+removeCert.message=Remove supporter certificate?
+removeCert.description=Are you sure you want to remove the certificate?
```

Comment:
Instead of asking the same question twice, add additional info here, e.g. "Even without a supperter cert Cryptomator stays functional and neither restrict access to your vaults nor lowers its security". A good example is the remove vault dialog.